<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMaiWindow</class>
 <widget class="QMainWindow" name="QMaiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassicalKitchen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Amethyst</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classical kitchen</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="randomBtn">
       <property name="font">
        <font>
         <family>Britannic Bold</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Random Recipe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="listBtn">
       <property name="font">
        <font>
         <family>Britannic Bold</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LIST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="font">
        <font>
         <family>Britannic Bold</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add Recipe</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
